pom.xml

```diff
@@ -103,7 +103,7 @@
             <dependency>
                 <groupId>com.typesafe.conductr</groupId>
                 <artifactId>${conductr.lib.name}</artifactId>
-                <version>1.4.7</version>
+                <version>1.9.1-SNAPSHOT</version>
```

Comment:
We have to publish a new `conductr-lib` with the aforementioned PR and update this reference.